apps/learner/src/routes/(protected)/(core)/learning/+page.server.ts

```diff
@@ -45,5 +45,6 @@ export const load: PageServerLoad = async (event) => {
 
   return {
     collections,
+    user: user.name,
```

Comment:
Do we have a better name for the key?

Reply:
Any suggestions? Else i'd update it to `username`